alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -15,6 +15,11 @@ const COLOR_FILL_ALPHA_STEP_3: Pixel = Pixel { _r: 64, _g: 64, _b: 64 };
 /// Default color used for filling.
 const COLOR_FILL: Pixel = Pixel { _r: 255, _g: 255, _b: 255 };
 
+const POWERLINE_TRIANGLE: char = '\u{e0b0}';
+const POWERLINE_ARROW: char = '\u{e0b1}';
+const POWERLINE_TRIANGLE_FLIPPED: char = '\u{e0b2}';
+const POWERLINE_ARROW_FLIPPED: char = '\u{e0b3}';
```

Comment:
Can we change this from flipped to RTL? RTL should be a pretty common abbreviation for text/unicode stuff and might be a little more descriptive than "flipped".